### PR TITLE
Closes #1649 Closes #1754 Support gcs files with spaces + be robust to workflow log deletion fa…

### LINKED ISSUE #1649: Cromwell does not exit in single workflow mode, using JES, on successful workflow completion
- JES backend

```
...snip....
[2016-11-03 19:36:22,19] [info] SingleWorkflowRunnerActor writing metadata to /home/lichtens/test_eval/crsp_validation_input_files/crsp_validation_from_cromwell.json.metadata.json
[2016-11-03 19:36:22,30] [info] SingleWorkflowRunnerActor workflow finished with status 'Succeeded'.
[2016-11-03 19:36:22,30] [info] WorkflowManagerActor: Received shutdown signal.
[2016-11-03 19:36:22,30] [info] Waiting for 1 workflows to abort...
....15 minutes gone by....
```

### LINKED ISSUE #1754: JES workflow fails with new version of cromwell
- JES backend
- 0.23
- single workflow

This workflow used to complete successfully (though cromwell did not exit), but with release 0.23, the workflow itself fails
Looks like cromwell can no longer handle spaces in the output file name.  I believe that @kshakir had a similar issue in one of the develop builds.  Did the fix make it into release 0.23?  

```
...snip...
java.lang.RuntimeException: Task 5d13ddf0-dcf9-4b99-bd13-40b4321a954a:aggregate_results_html failed: error code 5. Message: 9: Failed to localize files: failed to copy
the following files: "gs://broad-dsde-methods/cromwell-executions-eval-gatk-protected/crsp_validation_workflow/5d13ddf0-dcf9-4b99-bd13-40b4321a954a/call-run_plot_purity
_series/glob-2a33a5ba399f044203396c79c9f80928/purity_series_small_Small%20Amplifications.png -> /mnt/local-disk/broad-dsde-methods/cromwell-executions-eval-gatk-protect
ed/crsp_validation_workflow/5d13ddf0-dcf9-4b99-bd13-40b4321a954a/call-run_plot_purity_series/glob-2a33a5ba399f044203396c79c9f80928/purity_series_small_Small Amplificati
ons.png (cp failed: gsutil -q -m cp gs://broad-dsde-methods/cromwell-executions-eval-gatk-protected/crsp_validation_workflow/5d13ddf0-dcf9-4b99-bd13-40b4321a954a/call-r
un_plot_purity_series/glob-2a33a5ba399f044203396c79c9f80928/purity_series_small_Small%20Amplifications.png /mnt/local-disk/broad-dsde-methods/cromwell-executions-eval-g
atk-protected/crsp_validation_workflow/5d13ddf0-dcf9-4b99-bd13-40b4321a954a/call-run_plot_purity_series/glob-2a33a5ba399f044203396c79c9f80928/purity_series_small_Small
Amplifications.png, command failed: CommandException: No URLs matched: gs://broad-dsde-methods/cromwell-executions-eval-gatk-protected/crsp_validation_workflow/5d13ddf0
-dcf9-4b99-bd13-40b4321a954a/call-run_plot_purity_series/glob-2a33a5ba399f044203396c79c9f80928/purity_series_small_Small%20Amplifications.png\nCommandException: 1 file/
object could not be transferred.\n); gs://broad-dsde-methods/cromwell-executions-eval-gatk-protected/crsp_validation_workflow/5d13ddf0-dcf9-4b99-bd13-40b4321a954a/call-
run_plot_purity_series/glob-2a33a5ba399f044203396c79c9f80928/purity_series_small_Small%20Deletions.png -> /mnt/local-disk/broad-dsde-methods/cromwell-executions-eval-ga
tk-protected/crsp_validation_workflow/5d13ddf0-dcf9-4b99-bd13-40b4321a954a/call-run_plot_purity_series/glob-2a33a5ba399f044203396c79c9f80928/purity_series_small_Small D
eletions.png (cp failed: gsutil -q -m cp gs://broad-dsde-methods/cromwell-executions-eval-gatk-protected/crsp_validation_workflow/5d13ddf0-dcf9-4b99-bd13-40b4321a954a/c
all-run_plot_purity_series/glob-2a33a5ba399f044203396c79c9f80928/purity_series_small_Small%20Deletions.png /mnt/local-disk/broad-dsde-methods/cromwell-executions-eval-g
atk-protected/crsp_validation_workflow/5d13ddf0-dcf9-4b99-bd13-40b4321a954a/call-run_plot_purity_series/glob-2a33a5ba399f044203396c79c9f80928/purity_series_small_Small
Deletions.png, command failed: CommandException: No URLs matched: gs://broad-dsde-methods/cromwell-executions-eval-gatk-protected/crsp_validation_workflow/5d13ddf0-dcf9
-4b99-bd13-40b4321a954a/call-run_plot_purity_series/glob-2a33a5ba399f044203396c79c9f80928/purity_series_small_Small%20Deletions.png\nCommandException: 1 file/object cou
ld not be transferred.\n)"
        at cromwell.backend.impl.jes.JesAsyncBackendJobExecutionActor.cromwell$backend$impl$jes$JesAsyncBackendJobExecutionActor$$handleFailure(JesAsyncBackendJobExecut
ionActor.scala:530)
        at cromwell.backend.impl.jes.JesAsyncBackendJobExecutionActor$$anonfun$executionResult$1.apply(JesAsyncBackendJobExecutionActor.scala:560)
        at cromwell.backend.impl.jes.JesAsyncBackendJobExecutionActor$$anonfun$executionResult$1.apply(JesAsyncBackendJobExecutionActor.scala:537)
        at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)

...snip...
```

### PR DESCRIPTION
**Pull request by Horneth**
…ilures

### CODE REVIEW

File: core/src/main/scala/cromwell/core/logging/WorkflowLogger.scala [hunk 1]
```diff
@@ -109,7 +111,7 @@ class WorkflowLogger(loggerName: String,
 
   override def getName = loggerName
 
-  def deleteLogFile() = workflowLogPath foreach { File(_).delete(swallowIOExceptions = false) }
+  def deleteLogFile() = Try { workflowLogPath foreach { File(_).delete(swallowIOExceptions = false) } }
```
Comment (cjllanwarne) [inline core/src/main/scala/cromwell/core/logging/WorkflowLogger.scala]: Is this the same as just switching `swallowIOExceptions = true`?

**Comment by cjllanwarne** [comment]
👍

**Comment by ruchim** [comment]
👍

File: core/src/main/scala/cromwell/core/logging/WorkflowLogger.scala (hunk 1 above)
Comment (Horneth) [inline core/src/main/scala/cromwell/core/logging/WorkflowLogger.scala]: Yes except that the caller of this logs the error if the deletion failed

**Pull request merged**